test/built-ins/Promise/withResolvers/result.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2023 Peter Klecha. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Promise.withResolvers result is an object with keys "promise", "reject", and "resolve"
+esid: sec-promise.withresolvers
+features: [promise-with-resolvers, Object.hasOwn]
+---*/
+
+
+var instance = Promise.withResolvers();
+
+assert.sameValue(typeof instance, "object");
+assert.notSameValue(instance, null);
+assert(instance intanceof Object);
+assert(Object.hasOwn(instance, "promise"));
+assert(Object.hasOwn(instance, "resolve"));
+assert(Object.hasOwn(instance, "reject"));
```

Comment:
Would be nice to also test the property attributes here, using `verifyProperty()` from `propertyHelper.js`:

https://github.com/tc39/test262/blob/9d7eb8018e3b74ae9bcb12fbe8db0dfce4ef6adf/test/built-ins/BigInt/length.js#L27-L32

Reply:
(you can omit `value` as it's already tested elsewhere - https://github.com/tc39/test262/blob/9d7eb8018e3b74ae9bcb12fbe8db0dfce4ef6adf/harness/propertyHelper.js#L67-L71)